doc/devel/triage.rst

```diff
@@ -0,0 +1,214 @@
+    with "status: confirmed bug".
+
+  - **identify regressions**, determine if the reported bug used to
+    work as expected in a recent version of Matplotlib and if so
+    milestone for the next bug-fix release and label as "Release
+    critical".
+
+  - **close usage questions** and politely point the reporter to use
+    `discourse <https://discourse.matplotlib.org>`_ or Stack Overflow
+    instead and label as "community support".
+
+  - **close duplicate issues**, after checking that they are
+    indeed duplicate. Ideally, the original submitter moves the
+    discussion to the older, duplicate issue
+
+  - **close issues that cannot be replicated**, after leaving time (at
+    least a week) to add extra information
+
+
+
+.. topic:: Closing issues: a tough call
+
+    When uncertain on whether an issue should be closed or not, it is
+    best to strive for consensus with the original poster, and possibly
+    to seek relevant expertise. However, when the issue is a usage
+    question, or when it has been considered as unclear for many years it
+    should be closed.
+
+
+A typical workflow for triaging issues
+--------------------------------------
+
+The following workflow [1]_ is a good way to approach issue triaging:
+
+#. Thank the reporter for opening an issue
+
+   The issue tracker is many people’s first interaction with the
+   matplotlib project itself, beyond just using the library. As such,
+   we want it to be a welcoming, pleasant experience.
+
+#. Is this a usage question? If so close it with a polite message.
+
+#. Is the necessary information provided?
+
+   If crucial information (the version of Python, the version of
+   Matplotlib used, the OS, and the backend), is missing politely ask
+   the original poster to provide the information.
+
+#. Is the issue minimal and reproducible?
+
+   For bug reports, we ask that the reporter provide a minimal
+   reproducible example. See `this useful post
+   <https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports>`_
+   by Matthew Rocklin for a good explanation. If the example is not
+   reproducible, or if it's clearly not minimal, feel free to ask the reporter
+   if they can provide and example or simplify the provided one.
+   Do acknowledge that writing minimal reproducible examples is hard work.
+   If the reporter is struggling, you can try to write one yourself.
+
+   If a reproducible example is provided, but you see a simplification,
+   add your simpler reproducible example.
+
+   If you can not reproduce the issue, please report that along with your
+   OS, Pyhthon, and Matplotlib versions.
+
+   If we need more information from either this or the previous step
+   please label the issue with "status: needs clarification".
+
+#. Is this a regression?
+
+   While we strive for a bug-free library, regressions are the highest
+   priority.  If we have broken user-code that *used to* work, we should
+   fix that in the next patch release!
```

Comment:
Attempt to bisect here (if reproducer is simple, known good version exists, etc.)?

Reply:
Added paragraph below (so GH did not notice).